src/locales/ar.po

```diff
@@ -3136,83 +3108,101 @@ msgstr "الوصول/عدم الوصول"
 msgid "Predict whether or not the market will touch a chosen level before expiration."
 msgstr "توقع إذا كان السوق سيصل إلى مستوى مختار قبل انتهاء الصلاحية أم لا."
 
-msgid "asian down"
-msgstr "هبوط آسيوي"
+msgid "goes outside"
+msgstr "يخرج عن"
 
-msgid "Win payout if the last tick of %3 is strictly lower than the average of the %plural(%5,%d tick,%d ticks)."
-msgstr "أكسب عائد إذا ما كانت آخر علامة لل %3 هو أقل بدقة من المتوسط الجمعي(%5,%d علامة,%d علامات)."
+msgid "Win payout if %3 goes outside %7 to %6 through %5."
+msgstr "أربح عائد إذا %3 يذهب خارج %7 إلى %6 من خلال %5."
 
-msgid "asian up"
-msgstr "صعود آسيوي"
+msgid "Win payout if %3 goes outside %7 and %6 through %5 after %4."
+msgstr "أربح عائد إذا %3 يذهب خارج %7 و %6 من خلال %5 بعد %4."
 
-msgid "Win payout if the last tick of %3 is strictly higher than the average of the %plural(%5,%d tick,%d ticks)."
-msgstr "أكسب عائد إذا ما كانت آخر علامة لل %3 أعلى بدقة من المتوسط الجمعي(%5,%d علامة,%d علامات)."
+msgid "ends between"
+msgstr "ينتهي بين"
+
+msgid "Win payout if %3 ends strictly between %7 to %6 at %5."
+msgstr "أربح عائد إذا %3 ينتهي بدقة بين %7 إلى %6 عند %5."
+
+msgid "Win payout if %3 ends strictly between %7 to %6 at %5 after %4."
+msgstr "أربح عائد إذا %3 ينتهي بدقة بين %7 إلى %6 عند %5 بعد %4."
+
+msgid "spread down"
+msgstr "السبريد لأسفل"
+
+msgid "%1 %2 payout for every point %3 falls from entry level, with stop loss of %6 %4 and stop profit of %6 %5."
+msgstr "%1 %2 عائد عن كل نقطة %3 هبوط من مستوى الدخول، مع وقف خسارة %6 %4 ووقف ربح %6 %5."
+
+msgid "%1 %2 payout for every point %3 falls from entry level, with stop loss of %4 points and stop profit of %5 points."
+msgstr "%1 %2 عائد عن كل نقطة %3 هبوط من مستوى الدخول، مع وقف خسارة %4 نقاط ووقف ربح %5 نقاط."
+
+msgid "touches"
+msgstr "يصل إلى"
+
+msgid "Win payout if %3 touches %6 through %5."
+msgstr "أربح عائد إذا %3 يلامس %6 من خلال %5."
+
+msgid "Win payout if %3 touches %6 through %5 after %4."
+msgstr "أربح عائد إذا %3 يلامس %6 من خلال %5 بعد %4."
 
 msgid "higher"
 msgstr "أعلى"
 
-msgid "Win payout if %3 is strictly higher than %6 at %5."
-msgstr "أكسب عائد إذا ما كانت ال %3 أعلى بدقة من %6 عند %5."
+msgid "Win payout if %3 is higher than or equal to %6 at %5."
+msgstr "أربح عائد إذا %3 أعلى من أو يساوي %6 عند %5."
 
-msgid "Win payout if %3 is strictly higher than %6 at %5 after %4."
-msgstr "أكسب عائد إذا ما كانت ال %3 أعلى بدقة من %6 عند %5 بعد %4."
+msgid "Win payout if %3 is higher than or equal to %6 at %5 after %4."
+msgstr "أربح عائد إذا %3 أعلى من أو تساوي ل %6 عند %5 بعد %4."
 
-msgid "Win payout if %3 after %plural(%5,%d tick,%d ticks) is strictly higher than %6."
-msgstr "أكسب عائد إذا ما كانت ال %3 بعد % الجمعي(%5,%d علامة,%d علامات) بدقة أعلى من %6."
+msgid "Win payout if %3 after %plural(%5,%d tick,%d ticks) is higher than or equal to %6."
+msgstr "أربح عائد إذا %3 بعد % الجمع(%5, علامة دي%, علامات دي%) أعلى من أو تساوي ل %6."
```

Comment:
Further, Arabic has 6 plural forms according to this header:

```
"Plural-Forms: nplurals=6; plural=n==0 ? 0 : n==1 ? 1 : n==2 ? 2 : n%100>=3 && n%100<=10 ? 3 : n%100>=11 ? 4 : 5;\n"
```

The translator provides only 2.